theorems/T000473.md

```diff
@@ -9,4 +9,6 @@ refs:
     name: Infinite games and generalizations of first-countable spaces (Gruenhage)
 ---
 
-Asserted on page 340 of {{doi:10.1016/0016-660X(76)90024-6}}.
+Follows from Theorem 3.2 on page 342 of {{doi:10.1016/0016-660X(76)90024-6}}.
+Given a countable base $\{B_n:n<\omega\}$, the winning strategy for the first player is to
```

Comment:
I think it should say "Given a decreasing local base $\\{B_n:n<\omega\\}$ at a point $x$, ...".  Without decreasing it would not work.  And we don't assume second countable; only what's happening at each point matters.